packages/jest-snapshot/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jest-snapshot",
+  "version": "12.0.2",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "src/index.js",
+  "dependencies": {
+    "jest-util": "^12.0.2",
+    "react": "^15.0.2",
```

Comment:
I'll move react to devDependency for the test; about react-dom I'm using in the serializer logic, my plan involves making any serializer logic load through config, so that we can move the react-serializer to another project, removing any dependency.